x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/empty_list.tsx

```diff
@@ -20,46 +20,44 @@ export const EmptyList: FunctionComponent = () => {
   const history = useHistory() as ScopedHistory;
 
   return (
-    <EuiPageBody>
-      <EuiPageContent>
-        <EuiEmptyPrompt
-          iconType="managementApp"
-          data-test-subj="emptyList"
-          title={
-            <h2 data-test-subj="title">
-              {i18n.translate('xpack.ingestPipelines.list.table.emptyPromptTitle', {
-                defaultMessage: 'Start by creating a pipeline',
+    <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
+      <EuiEmptyPrompt
+        iconType="managementApp"
+        data-test-subj="emptyList"
+        title={
+          <h2 data-test-subj="title">
+            {i18n.translate('xpack.ingestPipelines.list.table.emptyPromptTitle', {
+              defaultMessage: 'Start by creating a pipeline',
+            })}
+          </h2>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.ingestPipelines.list.table.emptyPromptDescription"
+              defaultMessage="For example, you might create a pipeline with one processor that removes a field and another processor that renames a field."
+            />
+            <br />
```

Comment:
Maybe worth checking with design, but I'm not sure the line break is needed here anymore.

Reply:
I think you are right, most of the other apps just have the description + link without a line break. So let's keep it consistent!